IHP/IDE/SchemaDesigner/Controller/Columns.hs

```diff
@@ -54,6 +54,11 @@ instance Controller ColumnsController where
             let referenceTable = param "referenceTable"
             let onDelete = NoAction
             updateSchema (addForeignKeyConstraint tableName columnName constraintName referenceTable onDelete)
+
+            let indexName = tableName <> "_index"
```

Comment:
Creating `users.project_id` will make a index `users_index`. I think this is not unique enough (e.g. when you have two fks, both would be named `users_index`). It would make sense to name it like `users_project_id_index`

Reply:
Also just saw that `updateSchema` is called twice in the action, that triggers two file changes to `Schema.sql`. This triggers two app rebuilds. We should rather combine the two patch functions using `.` and then call `updateSchema` only once.

Reply:
I guess while we're on this we could also add some tests for this. I guess for that we might need to extract the domain logic a little bit out of the actions.